tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/validation/CurriculumMembershipValidator.java

```diff
@@ -131,6 +161,26 @@ private List<FieldError> checkCmWithPm(CurriculumMembershipDTO cmDto) {
     return fieldErrors;
   }
 
+  private List<FieldError> checkCmWithPmForBulk(CurriculumMembershipDTO cmDto) {
+    List<FieldError> fieldErrors = new ArrayList<>();
+    Optional<CurriculumMembership> dbCm = cmRepository.findById(cmDto.getId());
+    if (!dbCm.isPresent()) {
+      fieldErrors.add(new FieldError(CURRICULUM_MEMBERSHIP_DTO_NAME, FIELD_CURRICULUM_ID,
+          NO_MATCHING_CURRICULUM));
```

Comment:
I think the error here shoud be `NO_MATCHING_CURRICULUM_MEMBERSHIP`, and I cannot find any tests in validatorTest file for this error.